scripts/g.manual/g.manual.py

```diff
@@ -78,11 +78,19 @@ def start_browser(entry):
                 entry,
             )
     else:
-        path = os.path.join(gisbase, "docs", "html", entry + ".html")
-        if not os.path.exists(path) and os.getenv("GRASS_ADDON_BASE"):
-            path = os.path.join(
-                os.getenv("GRASS_ADDON_BASE"), "docs", "html", entry + ".html"
+        directory = os.path.join(gisbase, "docs", "mkdocs", "site")
+        if os.path.exists(directory):
+            path = os.path.join(directory, entry + ".html")
+            if not os.path.exists(path) and os.getenv("GRASS_ADDON_BASE"):
+                path = os.path.join(
+                os.getenv("GRASS_ADDON_BASE"), "docs", "mkdocs", "site", entry + ".html"
```

Comment:
We don't know what the path will be or even how we are going to handle this, but currently, HTML is present for g.extension installed addons, so you actually need the same URL as for HTML below.